### PR TITLE
Update README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pgonzal**
Improve the docs:
- Announce the new TSDoc playground (hurray!)
- Explain how to use Rush/Yarn
- Add a road map

### CODE REVIEW

File: api-demo/README.md [hunk 1]
```diff
@@ -3,37 +3,89 @@
...
+### Building and running api-demo api-demo (using Rush and Yarn)
+
+If you're going to submit a pull request for TSDoc, you will need to use the Rush monorepo
+manager tool.  See [Contributing.md](../Contributing.md) for more information.
+
+> Rush will link **api-demo** to use your local build of the **@microsoft/tsdoc** library,
+> for easy testing/validation.
+
+
+1. Install the [Rush](https://rushjs.io/pages/developer/new_developer/) software:
```
Comment (iclanton) [inline api-demo/README.md]: >software [](start = 72, length = 8)

*tool

File: api-demo/README.md [hunk 2]
```diff
@@ -3,37 +3,89 @@
...
+
+  *NOTE: If this command fails because your user account does not have permissions to
+  access NPM's global folder, you may need to
+  [fix your NPM configuration](https://docs.npmjs.com/getting-started/fixing-npm-permissions).*
+
+2. Install dependencies for all projects in the monorepo:
+
+  ```shell
+  # Run this command in the folder where you cloned the TSDoc repo from GitHub
+  $ rush install
```
Comment (iclanton) [inline api-demo/README.md]: That isn't really important.

File: Contributing.md [hunk 3]
```diff
@@ -57,6 +68,70 @@ Jest is configured to run using a plugin that adds TypeScript support.  You can
 the TypeScript source files, and the VS Code debugger will break on them.
 
 
+## Submitting a PR
+
+Before submitting your PR, you will need to install the Rush tool and build the monorepo.
+
+> **What's this Rush thing?**  Rush is Microsoft's build orchestrator that handles policy validation,
```
Comment (iclanton) [inline Contributing.md]: >Microsoft [](start = 39, length = 9)

It's not really specifically Microsoft's

File: api-demo/package.json [hunk 4]
```diff
@@ -17,6 +17,7 @@
   },
   "scripts": {
     "build": "rimraf ./lib/ && echo -- TYPESCRIPT -- && tsc && echo -- TSLINT -- && tslint --config tslint.json --project . --rules-dir node_modules/tslint-microsoft-contrib",
-    "start": "node ./lib/start.js"
+    "simple": "node ./lib/start.js simple",
+    "advanced": "node ./lib/start.js advanced"
   }
```
Comment (nickpape-msft) [inline api-demo/package.json]: in the tsdoc package.json, it looks like we aren't running tests

File: api-demo/README.md [hunk 5]
```diff
@@ -3,37 +3,89 @@
...
+   # Note: Be sure to specify "--no-package-lock" to avoid conflicts with Rush
+   $ npm install --no-package-lock
    ```
 
-3. Build the **api-demo** project like this:
+2. Build the **api-demo** project like this:
 
-   ```
+   ```shell
    $ npm run build
```
[the remark this answers is not in the document]
Comment (pgonzal) [reply api-demo/README.md]: I'll set this up later.

---
In reply to: [224282110](https://github.com/Microsoft/tsdoc/pull/105#discussion_r224282110) [](ancestors = 224282110)

File: api-demo/package.json (hunk 4 above)
Comment (pgonzal) [reply api-demo/package.json]: Good catch -- fixed

---
In reply to: [224280950](https://github.com/Microsoft/tsdoc/pull/105#discussion_r224280950) [](ancestors = 224280950)

File: tsdoc/build.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+try {
+  child_process.execSync(path.join(baseDir, 'node_modules/.bin/rimraf')
+    + ' ./lib/', { stdio: 'inherit' });
+
+  console.log('-- TYPESCRIPT --\n');
+  child_process.execSync(path.join(baseDir, 'node_modules/.bin/tsc'), { stdio: 'inherit' });
+
+  console.log('-- TSLINT --\n');
+  child_process.execSync(path.join(baseDir, 'node_modules/.bin/tslint')
+    + ' --config tslint.json --project . --rules-dir node_modules/tslint-microsoft-contrib',
```
Comment (iclanton) [inline tsdoc/build.js]: > --rules-dir node_modules/tslint-microsoft-contrib [](start = 40, length = 50)

This isn't in the tslint.json?

**Comment by iclanton** [comment]
> 

This should go at the end of the `try` block.

---
Refers to: tsdoc/build.js:39 in 9a4bab3. [](commit_id = 9a4bab363e52ad49f1cd6018a1115567cb002bb3, deletion_comment = False)

File: api-demo/build.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+try {
+  child_process.execSync(path.join(baseDir, 'node_modules/.bin/rimraf')
+    + ' ./lib/', { stdio: 'inherit' });
+
+  console.log('-- TYPESCRIPT --\n');
+  child_process.execSync(path.join(baseDir, 'node_modules/.bin/tsc'), { stdio: 'inherit' });
+
+  console.log('-- TSLINT --\n');
+  child_process.execSync(path.join(baseDir, 'node_modules/.bin/tslint')
+    + ' --config tslint.json --project . --rules-dir node_modules/tslint-microsoft-contrib',
```
Comment (iclanton) [inline api-demo/build.js]: >ir node_modules/tslint-microsoft-contr [](start = 50, length = 38)

same here.

**Pull request merged**